hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -614,6 +614,38 @@ If the following property is not set or set to `true`, the following exception w
 java.io.IOException: From option fs.s3a.aws.credentials.provider java.lang.ClassNotFoundException: Class CustomCredentialsProvider not found
 ```
 
+## S3 Authorization Using S3 Access Grants
+
+[S3 Access Grants](https://aws.amazon.com/s3/features/access-grants/) can be used to grant accesses to S3 data using IAM Principals.
```

Comment:
this might be a good time to move everything related into authentication into its own markdown file (authentication.md) and link that off index. 

this entry would be its own heading ## S3 Access grants with a <a name> reference so index generation will create a quick link